react/features/invite/components/DialInNumbersForm.js

```diff
@@ -392,11 +390,16 @@ class DialInNumbersForm extends Component {
  * @param {Object} state - The Redux state.
  * @private
  * @returns {{
+ *     _currentUserName: React.PropTypes.string,
  *     _dialIn: React.PropTypes.object
  * }}
  */
 function _mapStateToProps(state) {
+    const { name }
+        = getLocalParticipant(state['features/base/participants']);
+
     return {
+        _currentUserName: name,
```

Comment:
Let's rename the "_currentUserName" to "_localUserName" or "_localUserDisplayName"